src/main/scala/autoversion/AutoVersionPlugin.scala

```diff
@@ -49,12 +49,23 @@ object AutoVersionPlugin extends AutoPlugin {
   }
 
   private lazy val suggestBump = Def.task {
-    val commits = unreleasedCommits.value
-    val suggestedBumps =
+    val log = sbt.Keys.streams.value.log
+    val default = defaultBump.value
+    val suggestedBumps = {
+      val commits = unreleasedCommits.value
       commits.flatMap(_.suggestedBump(majorRegexes.value, minorRegexes.value, bugfixRegexes.value))
-    if (suggestedBumps.isEmpty)
-      throw new RuntimeException("No commit matches either patterns for bugfix, minor or major bumps !")
-    else suggestedBumps.max
+    }
+
+    if (suggestedBumps.nonEmpty) suggestedBumps.max
+    else default match {
+      case None => sys.error("No commit matches either patterns for bugfix, minor or major bumps !")
```

Comment:
small nitpicking, but I tend to find throwing the exception directly simpler than using `sys.error`, since there's no indirection and `sys.error` is not that often used in Scala codebases that I know of.

Reply:
Sure, I can switch to `throw`. I like `sys.error` for sbt plugins because it's more terse, but `throw` works great too.